py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -149,10 +135,9 @@ def aten_ops_embedding(
         name,
         input=args[1],
         weight=args[0],
-        max_norm=args_bounds_check(args, 2),
-        norm_type=args_bounds_check(args, 3),
-        scale_grad_by_freq=args_bounds_check(args, 4),
-        sparse=args_bounds_check(args, 5),
+        # args[2] is the padding index, which is useful for training only
+        scale_grad_by_freq=args_bounds_check(args, 3),
+        sparse=args_bounds_check(args, 4),
```

Comment:
Has the schema changed? I see below that` max_norm `and `norm_type` are not part of it?

Reply:
It seems so, as per this snippet from `native_functions.yaml`: https://github.com/pytorch/pytorch/blob/3a300ed84efa8e458df02ee4f92532667cefbe29/aten/src/ATen/native/native_functions.yaml#L2194-L2199

Reply:
The changes look good. Just that since `scale_grad_by_freq` and `sparse` are also not supported, should we not add a validator for them?